Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2408,6 +2413,36 @@ public bool FillMissingEpisodeNumbersFromPath(Episode episode, bool forceRefresh
                 episodeInfo = new Naming.TV.EpisodeInfo();
             }
 
+            try
+            {
+                var libraryOptions = GetLibraryOptions(episode);
+                if (libraryOptions.EnableEmbeddedEpisodeInfos && string.Equals(episodeInfo.Container, "mp4", StringComparison.OrdinalIgnoreCase))
+                {
+                    // Read from metadata
+                    var mediaInfo = _mediaEncoder.GetMediaInfo(new MediaInfoRequest
+                    {
+                        MediaSource = episode.GetMediaSources(false)[0],
+                        MediaType = DlnaProfileType.Video,
+                        ExtractChapters = false
```

Comment:
This should be false by default, no need to set it

Reply:
I removed the parameter in the funtion call